src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -175,15 +184,36 @@ def load_arguments(self, _):
         c.argument('container_name', container_name_type, id_part='child_name_2')
         c.argument('item_name', item_name_type, id_part='child_name_3')
 
-    for command in ['list', 'show-log-chain']:
-        with self.argument_context('backup recoverypoint ' + command) as c:
-            c.argument('vault_name', vault_name_type, id_part=None)
-            c.argument('start_date', type=datetime_type, help='The start date of the range in UTC (d-m-Y).')
-            c.argument('end_date', type=datetime_type, help='The end date of the range in UTC (d-m-Y).')
-            c.argument('backup_management_type', backup_management_type)
-            c.argument('container_name', container_name_type)
-            c.argument('workload_type', workload_type)
-            c.argument('use_secondary_region', action='store_true', help='Use this flag to list recoverypoints in secondary region.')
+    with self.argument_context('backup recoverypoint list') as c:
+        c.argument('vault_name', vault_name_type, id_part=None)
+        c.argument('start_date', type=datetime_type, help='The start date of the range in UTC (d-m-Y).')
+        c.argument('end_date', type=datetime_type, help='The end date of the range in UTC (d-m-Y).')
+        c.argument('backup_management_type', backup_management_type)
+        c.argument('container_name', container_name_type)
+        c.argument('workload_type', workload_type)
+        c.argument('use_secondary_region', action='store_true', help='Use this flag to list recoverypoints in secondary region.')
+        c.argument('is_ready_for_move', arg_type=get_three_state_flag(), help='Option to retrieve recoverypoints that are ready to be moved to destination-tier.')
```

Comment:
Write the help text in this fashion- "Use this flag to retrieve recoverypoints....."
check help text of other similar arguments for reference.

Reply:
Done